Anacapa_db/anacapa_QC_dada2.sh

```diff
@@ -142,44 +109,31 @@ fi
 ######
 
 #Check that user has all of the default flags set
-if [[ -e ${IN} && ! -z ${OUT} && -e ${DB} && ! -z ${ADPT} && ! -z ${ILLTYPE} ]];
+echo $IN
+echo $OUT
+echo $DB
+if [[ -e ${IN} && ! -z ${OUT} && -e ${DB} ]];
 then
   echo "Required Arguments Given"
   echo ""
 else
   echo "Required Arguments Missing:"
-  echo "check that you included arguments or correct paths for -i -o -d -a and -t"
+  echo "check that you included arguments or correct paths for -i -o -d -a"
   echo ""
   exit
 fi
 
 # location of the config and var files
 source $DB/scripts/anacapa_vars.sh  # edit to change variables and parameters
 source $DB/scripts/anacapa_config.sh # edit for proper configuration
-DEF_MIN_LENGTH="${DB}/metabarcode_loci_min_merge_length.txt"
-
-if [[ -e ${HPC_HEADER} ]];
-then
-  source ${HPC_HEADER}
-  echo ""
-else
-  source $DB/scripts/Hoffman2_HPC_header.sh
-  echo ""
-fi
-
-##load modules / software
-${MODULE_SOURCE} # use if you need to load modules from an HPC
-${FASTX_TOOLKIT} #load fastx_toolkit
-${ANACONDA_PYTHON} #load anaconda/python2-4.2
-${ATS} #load ATS, Hoffman2 specific module for managing submitted jobs.
 
 ###
 
 ################################
 # Preprocessing .fastq files
 ################################
-suffix1=R1_001.fastq
```

Comment:
I'm referring to the lines in this file where you changed the suffix from fastq to fastq.gz. Do you remember if there were any other files you had to modify for this change?